documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,256 @@
+
+
+.Migrating VMs with independent persistent disks from VMware to OCP-V fails
+
+Migrating VMs with independent persistent disks from VMware to OCP-V fails. link:https://issues.redhat.com/browse/MTV-993[(MTV-993)]
+
+
+.Migrating VMs from one OCP-V cluster to another using timeouts and fails
+
+When the time to transfer the disks of a VM exceeds the timeout of the Export API in {ocp-name} the migration fails. The timeout of the Export API in {ocp-name} is set to 2 hours by default. You can increase the timeout to resolve this issue. link:https://issues.redhat.com/browse/MTV-1052[(MTV-1052)]
+
+
+.Guest operating system from vSphere might be missing
+
+When vSphere does not receive updates about the guest operating system from the VMware tools, it considers the information about the guest operating system to be outdated and ceases to report it. When this occurs, {project-short} is unaware of the guest operating system of the VM and is unable to associate it with the appropriate virtual machine preference or {ocp-name} template. link:https://issues.redhat.com/browse/MTV-1046[(MTV-1046)]
+
+
+.{product-short} forklift-controller pod crashes when receiving a disk without a datastore
+
+In some rare cases, the VM may be configured with a disk that is placed on a datastore that is no longer available in vSphere. MTV gets such disks without their datastore. Previously, the `forklift-controller` crashed in this situation, rendering {project-short} unusable. In {project-short} 2.6.0, {project-short} presents a critical validation for VMs with such disks, and the `forklift-controller` no longer crashes. link:https://issues.redhat.com/browse/MTV-1029[(MTV-1029)]
+
+
+
+
+.Failure to migrate an image-based VM from {osp} to the `default` project
+
+The migration process fails when migrating an image-based VM from {osp} to the `default` project. link:https://issues.redhat.com/browse/MTV-964[(MTV-964)]
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12430275[Known Issues] in Jira.
+
+
+
+[id="resolved-issues-26_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+
+.Deleting a provider automatically also deletes the PV
+
+In the earlier versions of {product-short}, the PV was not removed when the OVA provider was deleted. This has been resolved in {product-short} 2.6.0, and the PV is automatically deleted when the OVA provider is deleted. link:https://issues.redhat.com/browse/MTV-848[(MTV-848)]
+
+
+.Fix for data being lost when migrating VMware VMs with snapshots
+
+In the earlier versions of {product-short}, when migrating a VM that has a snapshot from VMware, the VM that was created in {ocp-name} Virtualization contained the data in the snapshot but not the latest data of the running VM that was migrated. This has been resolved in {product-short} 2.6.0.  link:https://issues.redhat.com/browse/MTV-447[(MTV-447)]
```

Comment:
```suggestion
In the earlier versions of {product-short}, when migrating a VM that has a snapshot from VMware, the VM that was created in {ocp-name} Virtualization contained the data in the snapshot but not the latest data of the VM. This has been resolved in {product-short} 2.6.0.  link:https://issues.redhat.com/browse/MTV-447[(MTV-447)]
```
while re-reading this, the VM might not be running and we already say "when migrating a VM" - so suggesting a minor improvement here